File: php/create.php
<?php

require_once __DIR__ . '/Configurator.php';

$config = Configurator::createDefaultConfigurations();

$supported_signatures_algorithms = $config->getOption('supported_signatures_algorithms');
$claims = $config->getOption('claims');
$signature_headers = $config->getOption('signature_headers');
$signature_key = $config->getPrivateKey();

$signer = Jose\Signer::createSigner($supported_signatures_algorithms);
$creator = new Jose\JWTCreator($signer);

$token = $creator->sign(
  $claims, 
  $signature_headers, 
  $signature_key
);

print $token;

File: ruby/create.rb
require 'jwt'
require 'openssl'
require_relative 'configurator'

config = Configurator.create_default_configurations

claims = config['claims']
signature_headers = config['signature_headers']
private_key = config.private_key
token = JWT.encode(claims, private_key, 'RS256', signature_headers)
print token
